### ISSUE TITLE
Shift-tabbing to an HTML input of type file, nothing is read. IE Only

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2598]
**Reported by sshinney on 2012-08-14 18:56**
When using NVDA to tab (or use the 7-8-9 keys) through an HTML page if I come to an input element of type file everything works fine if I'm moving forward through the document. However, if I'm using shift-tab to move backward, it does not read normally. When the button is highlighted, it just says "Button", The text field part is treated like a regular text field.

I have seen this in IE only, Chrome seems to read just fine. I've also tried giving the element an ARIA-Label. Like everything else it works fine with tab but is not read by shift-tab. The label element associated with this input element is also not read.

Here is the HTML to build the element having the problem.

```
<label for="file">File Location</label>
<input type="file" name="file" size="80" maxlength="100" class="browse_btn">
```

**Comment by leonardder** [comment]
I can somehow reproduce this in IE11, at least it looks quite strange in browse mode. I ged File location: on one line, and an unlabelled button below.

**Comment by jcsteh** [comment]
P3 to investigate.

**Comment by Adriani90** [comment]
@jcsteh do you have any suggestions regarding this issue?

**Comment by Adriani90** [comment]
IE support has ended completely in 2022 and the browser is being removed automatically from machines that still have it after an Edge update in February 2023. I am closing this issue as abandoned, since I don't expect any further accessibility updates to IE or to IE mode in Edge. If you are having this issue in other browsers, please open a new issue with updated steps to reproduce and a minimal test case.
